goocanvas2.yaml

```diff
@@ -5,13 +5,16 @@ config-opts:
   - --disable-static
   - --enable-maintainer-mode
   - --disable-gtk-doc
+  - --with-pygi_overrides_dir=/app/lib/python3.7/site-packages/gi/overrides
 make-install-args:
   - girdir=/app/share/gir-1.0
   - typelibdir=/app/lib/girepository-1.0
 sources:
   - type: archive
     url: https://download.gnome.org/sources/goocanvas/2.0/goocanvas-2.0.4.tar.xz
     sha256: c728e2b7d4425ae81b54e1e07a3d3c8a4bd6377a63cffa43006045bceaa92e90
+  - type: patch
```

Comment:
You don't need this patch to build goocanvas. Something like
```

             "make-install-args" : [
                 "girdir=/app/share/gir-1.0",
                 "typelibdir=/app/lib/girepository-1.0"
             ]
```
should do the job.

Reply:
Never mind. Should have read what the patch do.

Reply:
> You don't need this patch to build goocanvas.

Without this patch, it will try to install `GooCanvas.py` into `/usr/lib/python3.7/site-packages/gi/overrides/` instead of `/app/lib/python3.7/site-packages/gi/overrides/`. And it will fail, because `/usr/` is real-only.

> Something like
> 
> ```
> "make-install-args" : [
>     "girdir=/app/share/gir-1.0",
>     "typelibdir=/app/lib/girepository-1.0"
> ]
> ```
> 
> should do the job.

It worked for **GNOME** `3.32`. For **GNOME** `3.34` it is simple not enough.